modules/core/02-client/keeper/keeper_test.go

```diff
@@ -130,14 +130,16 @@ func (suite *KeeperTestSuite) TestSetClientState() {
 }
 
 func (suite *KeeperTestSuite) TestSetCounterparty() {
-	counterpartyClientID := suite.keeper.GetCounterparty(suite.ctx, testClientID)
-	suite.Require().Equal("", counterpartyClientID, "Counterparty client ID is not empty")
-
-	testCounterpartyID := "counterparty-1"
-	suite.keeper.SetCounterparty(suite.ctx, testClientID, testCounterpartyID)
+	merklePathPrefix := commitmenttypes.NewMerklePath("ibc", "")
```

Comment:
This needs to change to bytes as mentioned by github. i believe there's a commit serdar has bumping the version. But you should probably just copy over those changes

Reply:
Is still wip, I'm having troubles fixing the commitmenttypes